java/core/src/main/java/sleeper/core/record/process/status/ProcessStatusUpdate.java

```diff
@@ -20,4 +20,12 @@
 public interface ProcessStatusUpdate {
 
     Instant getUpdateTime();
+
+    default boolean isStartOfRun() {
+        return false;
+    }
```

Comment:
Should the isStartOfRun method be just on ProcessRunStartedUpdate? It might make it a bit less ambiguous if we say every update that could possibly be the start of a run must implement that interface.